themes/bootstrap3/templates/search/home.phtml

```diff
@@ -7,7 +7,8 @@
 
   // Set default value if necessary:
   if (!isset($this->searchClassId)) {
-    $this->searchClassId = 'Solr';
+    $config = $this->config()->get('config');
+    $this->searchClassId = $config->Site->defaultSearchBackend ?? 'Solr';
```

Comment:
Is this another generic fix that we could address separately?

Reply:
Yes, indeed it is.

Reply:
I've plucked it out to dev and dev-9.0 and then merged back to here, so that's one file less in the PR. :-)